.pipelines/templates/add-arc-extension.yaml

```diff
@@ -0,0 +1,12 @@
+steps:  
+  - bash: |
+      ../../scripts/add-extension.sh
```

Comment:
The working directory for this step is the `repo.path`, so I think you should be able to just invoke the script by `./scripts/add-extension.sh`

Reply:
Good catch, will update